draft-ietf-rtcweb-jsep.xml

```diff
@@ -621,12 +621,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             attribute, it MUST first check to see if the original resolution
             meets the criteria specified in the attribute, and transmit it
             untouched if so. If the original resolution is too large for the
-            attribute criteria, the sender SHOULD apply downscaling to the
+            attribute criteria, the sender downscales the
             output of the MediaStreamTrack in order to satisfy the criteria.
             In rare cases, where a receiver requires a minimum resolution which
             is greater than the native resolution of the video, the
-            sender SHOULD apply upscaling in order to provide that resolution.
-            The sender SHOULD NOT apply upscaling in any other cases.
+            sender SHOULD apply upscaling in order to provide that resolution if
+            allowed by the upscale policy set by the Javascript and MUST not upscale if
+            the policy does not allow it.
+            The sender SHOULD NOT apply upscaling in any other cases and MUST
+            NOT if the JS policy does not allow it.
```

Comment:
Suggested rewrite:
"The sender MUST not apply upscaling if the upscaling policy as set by the JavaScript
forbids it. Otherwise, if the receiver requires ..." and then the original text.